pytest/lib/cluster.py

```diff
@@ -891,7 +871,7 @@ def spin_up_node_and_push(i, boot_key, boot_addr):
 
     return ret
 
-def start_bridge(start_local_ethereum=True, handle_contracts=True, handle_relays=True, config=None):
+def start_bridge(nodes, start_local_ethereum=True, handle_contracts=True, handle_relays=True, config=None):
```

Comment:
I'd love to delegate to bridge how to initialize itself by having a cluster. We may change in future the strategy inside of bridge lib and ask several nodes instead of `nodes[0]` to speed the things up and increase robustness.